test/integration/sequelize/drop.test.ts

```diff
@@ -41,28 +41,4 @@ describe('Sequelize#drop', () => {
     // drop both tables
     await sequelize.drop();
   });
-
-  it('supports truncating cyclic associations with { cascade: true }', async () => {
```

Comment:
I removed this since I didn't think it added too much to the suite (since we have an unit test), but I've added something similar back; https://github.com/sequelize/sequelize/pull/15235/commits/ccbdf1eaaedb58bf3c92732baf78e200f73f7c3e
But maybe I should include some assertions to ensure that the tables are actually dropped?

Reply:
> But maybe I should include some assertions to ensure that the tables are actually dropped?

That would be good

Note that the test is there because `sequelize.drop` changes its behavior when the cascade option is set